src/demo-app/a11y/dialog/dialog-address-form-a11y.html

```diff
@@ -40,6 +40,6 @@ <h2 mat-dialog-title>Company</h2>
 </mat-dialog-content>
 
 <mat-dialog-actions>
-  <button mat-raised-button color="primary" mat-dialog-close>Submit</button>
   <button mat-raised-button mat-dialog-close>Close</button>
+  <button mat-raised-button color="primary" mat-dialog-close>Submit</button>
```

Comment:
Can you add the attribute to this one. Also sorry, but I just remembered that we deprecated `cdk-focus-initial`.  `cdkFocusInitial` is the proper one.

Reply:
No problem at all